Makefile

```diff
@@ -55,15 +55,6 @@ export XC_ARCH
 ARCH:=${XC_OS}_${XC_ARCH}
 export ARCH
 
-ifeq (${BASEIMAGE}, )
-ifeq ($(ARCH),linux_arm64)
-  BASEIMAGE:=arm64v8/ubuntu:18.04
-else
-	BASEIMAGE:=ubuntu:16.04
-endif
-endif
-export BASEIMAGE
-
```

Comment:
Are we not going to support arm builds, or has it been handled elsewhere

Reply:
We will be building multi-arch builds using docker buildx github action. Will take that in separate PR.